src/Status.ts

```diff
@@ -192,6 +192,13 @@ export class Status {
         return new Status(new StatusConfiguration('/', 'In Progress', 'x', true, StatusType.IN_PROGRESS));
     }
 
+    /**
+     * A sample Non-Task status. Goes to NON_TASK when toggled.
+     */
+    static makeNonTask(): Status {
+        return new Status(new StatusConfiguration('Q', 'Non-Task', 'A', true, StatusType.NON_TASK));
+    }
+
```

Comment:
I created this for a test that I ended up binning - but it seemed worth keeping, for completion.